### PR TITLE
adding Dizipub & WoxeBot presence | Re-opened due to an error in pull#1660

### PR DESCRIPTION
**Pull request by snowy1000**
See the actual request here: https://github.com/PreMiD/Presences/pull/1660

### CODE REVIEW

File: websites/D/Dizipub/dist/metadata.json [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+    "author": {
+      "name": "Snowy ✨",
+      "id": "693445343332794408"
+    },
+    "service": "Dizipub",
+    "description": {
+      "en": "Dizipub is a website that you can watch your favorite series and films.",
+      "tr": "Dizipub, favori film ve dizilerinizi kolayca bulabileceğiniz bir film sitesidir."
+    },
+    "url": ["dizipub.net"],
```
Comment (Bas950) [inline websites/D/Dizipub/dist/metadata.json:11]: url shouldnt be in an array if its only 1

File: websites/D/Dizipub/dist/metadata.json [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+    "version": "1.0.0",
+    "logo": "http://snowy.is-online.me/i/5povs8eac4i1c2h.png",
+    "thumbnail": "http://snowy.is-online.me/i/cpplis87jiwtq2h.png",
+    "color": "#00BA66",
+    "tags": [],
```
Comment (Bas950) [inline websites/D/Dizipub/dist/metadata.json:16]: tags shouldnt be empty

Comment (snowy1000) [reply websites/D/Dizipub/dist/metadata.json]: Done.

File: websites/D/Dizipub/dist/metadata.json (hunk 1 above)
Comment (snowy1000) [reply websites/D/Dizipub/dist/metadata.json]: Done.

**Comment by snowy1000** [comment]
Also adding WoxeBot presence on commit 358c5a3, re-requesting review. 
TJwmn5270qyk
22iSGNbmyyKB
EUK8xcffxseJ
PrSTJ3fefspt

**Pull request merged**